frontend/src/metabase/visualizations/lib/apply_axis.js

```diff
@@ -285,7 +285,8 @@ export function applyChartQuantitativeXAxis(
   }
 
   // pad the domain slightly to prevent clipping
-  xDomain = [xDomain[0] - xInterval * 0.75, xDomain[1] + xInterval * 0.75];
+  const pad = Math.round(xInterval * 0.75 * 10) / 10;
```

Comment:
@iethree works nice! I'm wondering why not simply `const pad = Math.round(xInterval * 0.75);`

Reply:
Good question! I wanted to preserve a bit more precision than nearest whole number. If we're looking at numbers less than 1, rounding to a whole number would round the pad to 0, giving us graphs that overlap the vertical axes like this:

![Screenshot from 2022-03-28 15-25-07](https://user-images.githubusercontent.com/30528226/160490042-8c8ef0ec-8bbd-4d64-9f53-39f0b30d886b.png)

Which is exactly what this padding is trying to eliminate

Reply:
Good, thanks for your clarification!